src/models/thread.ts

```diff
@@ -317,15 +325,18 @@ export class Thread extends ReadReceipt<EmittedEvents, EventHandlerMap> {
         }
         this.lastPendingEvent = pendingEvents.length ? pendingEvents[pendingEvents.length - 1] : undefined;
         this.pendingReplyCount = pendingEvents.length;
+    }
 
-        if (Thread.hasServerSideSupport && bundledRelationship) {
-            this.replyCount = bundledRelationship.count;
-            this._currentUserParticipated = !!bundledRelationship.current_user_participated;
-
-            const mapper = this.client.getEventMapper();
-            this.lastEvent = mapper(bundledRelationship.latest_event);
-            await this.processEvent(this.lastEvent);
+    private async updateThreadMetadata(): Promise<void> {
```

Comment:
was this meant to be turned private? initialiseThread() was previously used if somehow the threadtimeline hasn't loaded when requested so was called then. Is there another way to force a fetch/updating the thread data since `updateThreadMetadata` is now private?

Reply:
updateThreadMetadata shouldn't be loading the thread timeline in the first place, that timeline should be handled the same way as all other timelines nowadays. You should be able to just use `client.getLatestTimeline()` on the threads' timelineset, or `client.paginateEventTimeline()` on the threads' liveTimeline to force it to load.